nav2_bringup/launch/nav2_params.yaml

```diff
@@ -39,6 +40,13 @@ DwbController:
 local_costmap:
   local_costmap:
     ros__parameters:
+      global_frame: base_link
```

Comment:
Shouldn't the global frame be `odom` and the robot base frame be `base_link`? This would be the same as the rolling local costmap setup for ROS1.

Reply:
Yes. I think you are right.

I originally wanted to make this work because of the "robot centric costmap" that was added in #516. I didn't want to have 3 costmaps in the system if it could be avoided. I was hoping DWB and the motion primitives could share the same one. 

However, I don't think costmap is built to really support a rapidly moving reference frame. I think this means the robot centric costmap in that PR should really be referenced to the odom frame as well.